com.one_ware.OneWare.yml

```diff
@@ -0,0 +1,49 @@
+app-id: com.one_ware.OneWare
+runtime: org.freedesktop.Platform
+runtime-version: '23.08'
+sdk: org.freedesktop.Sdk
+command: oneware
+separate-locales: false
+tags:
+  - proprietary
+finish-args:
+  - --persist=.local/share/OneWareStudio
+  - --device=all
```

Comment:
why do you need `all` here?

Reply:
I need access to USB devices for programming the hardware and using components like Serial Monitor.
In the [docs](https://docs.flatpak.org/en/latest/sandbox-permissions.html#device-access) it was not clear to me if dri and input would be enough

Reply:
all is ok, input is not widely available.